README.md

```diff
@@ -48,14 +48,16 @@ limitations under the License.
 </h4>
 
 <h3 align="center">
-    <p>State-of-the-art Natural Language Processing for Jax, PyTorch and TensorFlow</p>
+    <p>State-of-the-art Machine Learning for JAX, PyTorch and TensorFlow</p>
 </h3>
 
 <h3 align="center">
     <a href="https://hf.co/course"><img src="https://raw.githubusercontent.com/huggingface/transformers/master/docs/source/imgs/course_banner.png"></a>
 </h3>
 
-🤗 Transformers provides thousands of pretrained models to perform tasks on texts such as classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. Its aim is to make cutting-edge NLP easier to use for everyone.
+🤗 Transformers provides thousands of pretrained models to perform tasks on different modalities such as language, vision, and audio. 
+
+These models can perform tasks on texts such as text classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. They can perform tasks on images such as image classification, object detection, segmentation. They can perform tasks on audio such as speech recognition and audio classification. They can perform tasks on several modalities combined, such as table question answering, optical character recognition, information extraction from scanned documents, video classification, visual question answering.
```

Comment:
This paragraph has a lot of repeated "They can perform tasks". What do you think about rephrasing it a bit like the following? (Just an idea)

```suggestion
These models can applied on:

* 📝 Text, for text classification, information extraction, question answering, summarization, translation, text generation, and more in over 100 languages. 
* 🖼️ Images, for image classification, object detection, and segmentation. 
* 🗣️ Audio, for speech recognition and audio classification. 

Transformer models can also perform tasks on **several modalities combined**, such as table question answering, optical character recognition, information extraction from scanned documents, video classification, and visual question answering.
```

Reply:
I love it! The only issue I see is that with the original phrasing, it's open-ended: "tasks like ..." which also mean that tasks not included here are also handled (which is the case!). In the proposed wording, it seems like it's limited to the task mentioned. If you can rephrase your proposal to take this into account, I'm happy to merge it!

Reply:
I tweaked the text to include "for tasks such as" in each bullet point. It's a bit repetitive, but maybe the use of a list and emojis is the main improvement here :)